drivers/bridge/bridge.go

```diff
@@ -199,7 +220,7 @@ func (d *driver) DeleteNetwork(nid driverapi.UUID) error {
 	return err
 }
 
-func (d *driver) CreateEndpoint(nid, eid driverapi.UUID, sboxKey string, config interface{}) (*driverapi.SandboxInfo, error) {
+func (d *driver) CreateEndpoint(nid, eid driverapi.UUID, sboxKey string, epOption interface{}) (*driverapi.SandboxInfo, error) {
```

Comment:
Shall we use a consistent naming for the options variable between CreateNetwork, CreateNetworkDriver and CreateEndpoint ?